### PR TITLE
fix: fix ci tool

### PR DESCRIPTION
**Pull request by haiyux**
#### Which issue(s) this PR fixes (resolves / be part of):

fixes #

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -60,7 +60,10 @@ linters-settings:
     settings:
       mnd:
         # don't include the "operation", "argument" and "assign"
-        checks: case,condition,return
+        checks: 
+          - case
+          - condition
+          - return
```
Comment (kagaya85) [inline .golangci.yml:66]: Is this configuration valid? It seems that not appear in the official gomnd [docs](https://golangci-lint.run/usage/linters/#gomnd)

Comment (haiyux) [reply .golangci.yml:66]: image

Comment (haiyux) [reply .golangci.yml]: https://github.com/golangci/golangci-lint/pull/2498/files

**Review by kagaya85 — APPROVED** [review]
LGTM

File: .golangci.yml (hunk 1 above)
Comment (kagaya85) [reply .golangci.yml]: ok, got it

**Review by tonybase — APPROVED** [review]

**Pull request merged**